pyproject.toml

```diff
@@ -162,18 +121,22 @@ inline-quotes = "double"
 multiline-quotes = "double"
 
 [tool.pixi.workspace]
-channels = ["conda-forge"]
-platforms = [
-  "linux-64",
-  "osx-64",
-  "osx-arm64",
-  # The google-cloud-sdk conda package doesn't support Windows
-  # "win-64",
+channels = [
+  "https://prefix.dev/conda-forge",
+  "https://prefix.dev/pixi-build-backends",
```

Comment:
Is `pixi-build-backends` still required? I saw it was used in the [issue](https://github.com/prefix-dev/pixi/issues/1569) but also the comments implied that it was only temporarily required before the python-build-backend made it to conda-forge.